app/themes/dashboard/ui/styles/theme.less

```diff
@@ -43,3 +43,11 @@ z-index: 10;
 .pre-xml {
   background-color: #ecf0f5;
 }
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
+}
+
+.nav-pills>li.active>a {
+  background-color: #384246 !important;
```

Comment:
Well, this bit is in theme.less, so won't interfere, but I agree we should try to avoid !important if possible.

By the way, if the !important was necessary to override something from main.less, I will probably be moving the contents of main.less into a separate import that will be loaded *before* theme.less. I think that should make it easier to override our default designs..